### ISSUE TITLE
`</script>` → `<\/script>` workaround

### ISSUE THREAD

**Issue by mathiasbynens** [remy/inliner#11]
You have [a workaround](https://github.com/remy/inliner/blob/ffc6431b0ea4350f8331e2ef25ba771454f0f2c2/inliner.js#L215-216) in place an uglify.js bug, which is great. Might be interesting to keep an eye on https://github.com/mishoo/UglifyJS/issues/164 (I’m still hoping this will be fixed in uglify.js itself).

I recently did some ETAGO research, and you can read the full details here: http://mathiasbynens.be/notes/etago Here’s the TL;DR:

> Just keep in mind that the full `</style` and `</script` strings followed by a space character, `>`, or `/` will close their respective opening tag.

In other words, the current workaround only detects one of those three cases.

**Comment by remy** [comment]
This should be fixed @ 1.0.0 (but will reopen otherwise!).
